fec/fec/static/js/calendar.js

```diff
@@ -339,6 +348,27 @@ Calendar.prototype.handleEventClick = function(calEvent, jsEvent, view) {
   }
 };
 
+Calendar.prototype.handleKeypress = function(e) {
+  if (e.keyCode === 13) {
+    $(e.target).click();
+  }
+};
+
+Calendar.prototype.controlPopoverFocus = function(e) {
```

Comment:
This method does things other than focus. Can we give it a more descriptive name? A docstring might be useful here.